js/ui/scheduler/workspaces/ui.scheduler.timeline.js

```diff
@@ -91,6 +91,14 @@ var SchedulerTimeline = SchedulerWorkSpace.inherit({
         return this._getCellCountInDay() * this.option("intervalCount");
     },
 
+    getGroupTableWidth: function() {
+        if(this._$sidebarTable) {
```

Comment:
I suggest making this "if" operator more compactly:  `this._$sidebarTable ? this._$sidebarTable.outerWidth() : 0 `